src/server/util/Util.js

```diff
@@ -16,6 +16,25 @@ function post(req, callback) {
     }
 }
 
+function get(req, callback) {
+    if(req.method === 'GET') {
+        let body = ''
+        
+        req.on('data', chunk => {
```

Comment:
Ideally GET requests must not contain any data in the body. So this part of code is redundant.